op-chain-ops/deployer/opcm/opchain.go

```diff
@@ -34,6 +34,7 @@ type DeployOPChainInput struct {
 	BlobBaseFeeScalar uint32
 	L2ChainId         *big.Int
 	OpcmProxy         common.Address
+	SaltMixer         string
```

Comment:
op-deployer currently generates a `create2` salt on startup - could you wire this up with that?

Reply:
Using `st.Create2Salt` as the initial value of `SaltMixer`: https://github.com/ethereum-optimism/optimism/pull/12152/commits/093331de1baf4f610e8169e73950c3490e557fc3